conf/machine/include/tegra-common.inc

```diff
@@ -104,3 +105,13 @@ TEGRA_FLASHVARS ?= "BPFDTB_FILE BPF_FILE BR_CMD_CONFIG DEVICEPROD_CONFIG DEVICE_
 EMMC_BCTS ?= "${EMMC_BCT}${@',' + d.getVar('EMMC_BCT_OVERRIDE') if d.getVar('EMMC_BCT_OVERRIDE') else ''}"
 
 TEGRA_UEFI_SIGNING_CLASS ?= "tegra-uefi-signing"
+
+USE_REDUNDANT_FLASH_LAYOUT ?= "1"
```

Comment:
Hmm.  Do we want this to be the default for everyone?

Reply:
I probably wouldn't have been brave enough to try it but I did based on https://github.com/OE4T/meta-tegra/issues/1389 and the comment about setting as default.  It seems a bit scary but if you are tegraflashing and your rootfs is small enough to fit it seems like a low risk.  My guess is few people use the default flash layouts for production too, but perhaps I'm wrong about that.  Added more about this in https://github.com/OE4T/meta-tegra/pull/1428/commits/e0e04e85d4acd1c34b55620d749d3f5180b14ac7 as well.

Reply:
Good idea, I did that at https://github.com/OE4T/tegra-demo-distro/pull/284